plugins/hls-class-plugin/src/Ide/Plugin/Class/Types.hs

```diff
@@ -1,11 +1,11 @@
+{-# LANGUAGE CPP              #-}
```

Comment:
Change unrelated to eval plugin, but pre-commit check (which check styling-haskell formatting) was failing after [this PR](https://github.com/haskell/haskell-language-server/pull/3973) was merged to master.